manifests/r/redfox/AnyDVDHD/8.6.3.0/redfox.AnyDVDHD.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: redfox.AnyDVDHD
+PackageVersion: 8.6.3.0
+PackageLocale: en-US
+Publisher: RedFox
+PackageName: AnyDVD HD

```

```suggestion
PackageName: AnyDVD
```
This must match Control Panel.